test/test_ops.py

```diff
@@ -2083,7 +2083,7 @@ def test_tags(self, device, dtype, op):
             if isinstance(input, torch.Tensor):
                 old_size = input.size()
                 old_stride = input.stride()
-                with TestTagsMode():
+                with _TestTagsMode():
```

Comment:
Why rename this?

Reply:
When initiating this test, `pytest` complains that this class can't be parsed...which makes sense, it's a context manager. The name apparently makes it automatically parsed. Since it was only used in this file, I figured renaming it to silence an annoying warning was reasonable.